components/online_image.rst

```diff
@@ -32,13 +32,16 @@ This component has a dependency to :doc:`/components/http_request`; the configur
 
     online_image:
       - url: "https://example.com/example.png"
+        request_headers:
+          Authorization: !secret bearer_api_token
```

Comment:
The main example should be kept to a minimum as people (for some reason) like to copy paste it directly to their YAML, and we would like them to read and understand the configuration options instead.
```suggestion
```